roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -32,7 +32,7 @@
         - iptables.service
         - ip6tables.service
       when:
-        - ansible_facts.services["{{ item }}"] is defined
```

Comment:
need to keep "is defined" condition on top else it would have issues when service itself don't exist, same at other places